cms/templates/js/course-outline.underscore

```diff
@@ -147,7 +147,7 @@ if (is_proctored_exam) {
                                 {
                                     'service': 'studio',
                                     'screen_name': 'course_outline',
-                                    'organization': <%- platform_name %>,
+                                    'organization': '${ platform_name }',
```

Comment:
Can you explain this change?

Reply:
This syntax was causing some issue. The event was not being triggered previously